### ISSUE TITLE
Add a matching Menu Bar to the Community Forum for consistency

### ISSUE THREAD

**Issue by polterguess** [humanetech-community/community-hub#57]
Hopefully, this are just a few linked buttons.
Suggestions
1. Problem: The forum left logo goes to forum. The button for the home page is way on the right and it's very much no clear as a globe. 
  Solution: Instead change logo to go to homepage and then add a "Forum" or "Forum Home" button that links to the Forum homepage. 
2. Change the top menu of the forum to match the top menu of the homepage.
Example: Forum, Blog, Docs, Activities, Resources, About

**Comment by aschrijver** [comment]
Good feedback this. Especially 2). I have seen Discourse forums that had this, so should be possible. I will investigate.

**Comment by aschrijver** [comment]
Yes, I just tested a `theme-component` that adds a menu bar. This is how it looks like out of the box:

![humanetech-forum-menu-bar](https://user-images.githubusercontent.com/5111931/56740718-c1173600-6771-11e9-82df-37efe830aefb.png)

Some styling can be added to change colors, plus all entries can have icons. To have the styles exactly match that of the Community Hub website some more tweaking is needed. I don't know how well the CSS style overrides work for theme components, but there should we look for solutions.

**Comment by aschrijver** [comment]
@micheleminno FYI you can find the theme-component in the Admin > Customize section and then click on components. This component is currently not assigned to a theme (which makes it display immediately)

**Comment by micheleminno** [comment]
I should not add the menu bar to the forum, it would create confusion: which os the role of the forum? Which is the role of the website instead?
I would:
- replace the left logo on the forum with something like this:

![Screenshot 2019-04-26 at 11 03 38](https://user-images.githubusercontent.com/832220/56796509-f889ef00-6812-11e9-847c-1be6a7b7b323.png)

- replace the globe with a new icon representing the website:

![Screenshot 2019-04-26 at 10 46 01](https://user-images.githubusercontent.com/832220/56795302-81535b80-6810-11e9-942b-d536b7b30a3b.png)


- remove all other icons (twitter - it's now part of the social media channels in the website and github - too technical for most of the users, it would be linked from the be a builder page for active participation).

**Comment by aschrijver** [comment]
I don't think it creates this confusion. It makes it clear that the forum is sub-part of the Community Hub, and allows quick navigation to website sections (The menu can also hold some other forum links). Without having prominent links to the Community Hub, it will be a forgotten website and only some die-hards now and then contributing to it.

The globe icon cannot be replaced with that logo. The theme component only accepts `font-awesome` icons. Removing Twitter from it is a good idea.

I find it important to have the Github icon. I always look for this icon when I land on a project or community. For technical people it is important to see that stuff is OSS and on Github. Non-technical people can be involved in Issue tracker communication (i.e. for their own blog post contribution).

**Comment by micheleminno** [comment]
According to me, the forum must stay as plain and simple as possible, without navigation bar.
Non technical people don’t even know what is it and how to use an issue tracker.

**Comment by aschrijver** [comment]
> Non technical people don’t even know what is it and how to use an issue tracker.

The navigation in the menu is not to Github, but to content sections that exist on the Website.

> I should not add the menu bar to the forum, it would create confusion: which os the role of the forum? Which is the role of the website instead?

The website is where the HTC outputs her 'results' i.e. the content on Awareness and Solutions targeted at the broader public. The Community Hub is the new first point of entry to our community. The discussion forum (which has been primary point of access unitl now) now comes secondary and is for the purpose of discussions that ultimately lead to improvements to the Community Hub.

If we don't make it clear that the Community Hub is leading, then the forum continues to be the only interaction point with HTC for most members and readers. We continue to have 'just' discussions, that then sink out of sight somewhere deep down the list of topics.

Hence the menu: Show the same navigation that is on top of the Community Hub also on top of the forum.
